src/components/download_buttons/DownloadButtons.module.scss

```
@@ -1,35 +1,87 @@
 @import '../../utils/variables';
 
 .downloadBtns {
-  display: flex;
-  margin: 2em 0;
-  justify-content: space-between;
-  max-width: 600px;
-  flex-wrap: wrap;
-  & a {
-    display: flex;
-    width: 48%;
-    max-width: 300px;
-    max-height: 90px;
-  }
-  & img {
-    max-width: 100%;
-    width: 90%;
-  }
-}
-
-@media (max-width: $small) {
-  .downloadBtns {
+    margin: 2em 0;
+    height: 90px;
     & a {
```

I know this isn't part of your commit - but we don't actually need these ampersands
`& a` is the same as just having `a` (when nested and with a space) and I'd probably say not having them is a bit neater.
```suggestion
    a {
```
https://css-tricks.com/the-sass-ampersand/